training/src/main/resources/spring/camel-context.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="
+       http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans.xsd
+       http://camel.apache.org/schema/spring       http://camel.apache.org/schema/spring/camel-spring.xsd">
+
+  <!-- You could also define a traditional camel context here -->
+  <!--
+  <camelContext xmlns="http://camel.apache.org/schema/spring">
+      <route>
+          <from uri="timer://foo?period=1000"/>
+          <setBody><simple>Hello World from camel-contex.xml</simple></setBody>
+          <log message=">>> ${body}"/>
+      </route>
+  </camelContext>
+  -->
+</beans>
```

Comment:
I think this is not necessary because you are using [Java DSL](https://camel.apache.org/java-dsl.html)